# Generated for NaoQi method ALAudioDevice.stopMicrophonesRecording
# This method stops the recording of the sound collected by the microphones.

---


